client/src/components/layout/Votes.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from "react"
+
+const Votes = (props) => {
+  const [userVote, setUserVote] = useState({
+    vote: 0,
+    voted: false,
+  })
+  const [totalVotes, setTotalVotes] = useState(0)
+
+  const getVote = async () => {
```

Comment:
If Votes are associated with a Trip then we don't need this fetch request to get the current votes, instead we should make use of the serializers on the backend such that when we query for a given Trip is also provides us with its Votes